src/main/java/gov/nih/nci/bento/graphql/BuildBentoGraphQL.java

```diff
@@ -0,0 +1,126 @@
+package gov.nih.nci.bento.graphql;
+
+import gov.nih.nci.bento.model.AbstractESDataFetcher;
+import gov.nih.nci.bento.model.PrivateNeo4jDataFetcher;
+import graphql.GraphQL;
+import graphql.schema.GraphQLNamedType;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import org.neo4j.graphql.SchemaBuilder;
+import org.neo4j.graphql.SchemaConfig;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+
+public class BuildBentoGraphQL {
```

Comment:
Other project will use this class too. I think we should better rename it without Bento